Docker/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:trusty
+
+RUN apt-get update
+RUN apt-get -y install software-properties-common python-software-properties
+RUN add-apt-repository ppa:git-core/ppa
+RUN apt-get -y install git
+RUN apt-get -y install sudo
+RUN apt-get -y install wget
+
+RUN git clone -b docker https://github.com/JGulbronson/rmc.git ~/rmc
```

Comment:
I think I'm getting confused here, but won't this always run master? (Ignoring the fact that this should be UWflow/rmc.git, which I tweaked in https://github.com/UWFlow/rmc/pull/281).

If it's always running master, that means that if I open a PR that's completely broken, it'll still pass because master is passing, right?